### ISSUE TITLE
Textfield issues : when you type on the 'title' field, it bounces between "lastSavedValue" and "value" and desync from "TitleEditor"

### ISSUE THREAD

**Issue by zedee** [area17/twill#408]
I'm trying to change a "Title" field, which gets in sync with the slug, but it reacts weirdly when I type some text. I'm talking about the text field where is in sync with the title over the slug.

Inspecting with Vue extension, I see that every time you type on the text field, it bounces the value between this two data variables: `lastSavedValue` and `value`. 

Has anyone being able to reproduce this issue? I've managed to stop the weird behavior erasing `this.lastSavedValue = this.value` on `updateAndSaveValue` method, but I'm pretty sure that's not the idea.

*UPDATE*: As I have difficulties to explian correctly the issue, I've recorded a video of what's going on: https://drive.google.com/file/d/1n4oGlsfPkn73F7NyWvKxNMii44QME3v0/view

Please, could someone give some advice? I will do the PR by myself but I need some directions and more information about why the're some lines like these on `form.js` store:

```
// (Inside [FORM.UPDATE_FORM_FIELD]) 
// Update existing form field
    if (fieldIndex !== -1) {
      if (field.locale) fieldValue = state.fields[fieldIndex].value
      // remove existing field
      state.fields.splice(fieldIndex, 1)
    }
```

I don't know why code is so complicated in order to sync the components (all these event emitters on `Textfield.vue` ... Why not just syncing data to store from the input model, then on the `TitleEditor` just `compute` its value?

Something is out of sync here, but can't tackle the source, not even with the debugger. Please help! I'd like to fix by myself this bug, but I don't know where to start :(

**Comment by zedee** [comment]
Is this behavior only happening to me?

**Comment by zedee** [comment]
@yanhao-li @ifox 
Do you have any idea why is this happening? Or I'm the only one who has the issue?

**Comment by yanhao-li** [comment]
Hi @zedee ,

Usually It's cause by having two fields share same name on the same form, could you check if that happens on you?

I think we have a PR for supporting this but haven't been merged yet: https://github.com/area17/twill/pull/381
Would also be very helpful if you could help review that PR and verify if it's working.

Thank you!

**Comment by Insperio** [comment]
Same problem here. Just tested the PR you mentioned.  It fixes the problem

**Comment by zedee** [comment]
> Hi @zedee ,
> 
> Usually It's cause by having two fields share same name on the same form, could you check if that happens on you?
> 
> I think we have a PR for supporting this but haven't been merged yet: #381
> Would also be very helpful if you could help review that PR and verify if it's working.
> 
> Thank you!

Hi @yanhao-li 
I also tested the PR and merged into my fork. The issue is fixed now :)

Many thanks!!!
